### ISSUE TITLE
This site seems to detect it

### ISSUE THREAD

**Issue by FabriceSalvaire** [ultrafunkamsterdam/undetected-chromedriver#1740]
url is `l e r o y m e r l i n . f r`
I opened chrome and managed to connect to my account.
But quickly it demands to solve a puzzle and we are blocked
So I deleted the selenium directory and retried.
But now way, just after to solve the puzzle, same behaviour.
I guess it detects something.

**Comment by userbox020** [comment]
yes, seems like its been detected and its not bypassing captchas anymore
